tests/modules/services/window-managers/sway/sway-followmouse-legacy.nix

```diff
@@ -0,0 +1,33 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  config = {
+    wayland.windowManager.sway = {
+      enable = true;
+
+      config = {
+        focus.followMouse = false;
+        menu = "${pkgs.dmenu}/bin/dmenu_run";
+        bars = [ ];
+      };
+    };
+
+    nixpkgs.overlays = [
+      (self: super: {
+        dmenu = super.dmenu // { outPath = "@dmenu@"; };
+        rxvt-unicode-unwrapped = super.rxvt-unicode-unwrapped // {
+          outPath = "@rxvt-unicode-unwrapped@";
+        };
+        sway = super.sway // { outPath = "@sway@"; };
```

Comment:
Hopefully the last nitpick, but I don't think you use this now you have `bars = [ ]`

Reply:
If I remove `bars = []` then it outputs the default bar config with i3status and swaybar embedded (so I'd have to add overrides for those packages now). But I can't do that either, since the swaybar comes from the overriden sway package, I can't override the output path to fake it, so I'd have to set the bar's swaybarCommand, and then I'm in a worse place than I started... :)